backend/package.json

```diff
@@ -14,16 +14,18 @@
 	],
 	"activationEvents": [
 		"*",
-		"onCommand:sap.loadYeomanUI"
+		"onCommand:loadYeomanUI_projects",
+		"onCommand:loadYeomanUI"
 	],
 	"main": "./dist/extension",
 	"contributes": {
-		"commands": [
-			{
-				"command": "sap.loadYeomanUI",
-				"title": "Yeoman User Interface"
-			}
-		]
+		"commands": [{
+			"command": "loadYeomanUI_projects",
+			"title": "Yeoman UI Project Generators"
+		}, {
+			"command": "loadYeomanUI",
+			"title": "Yeoman UI Generators"
```

Comment:
"title": "Yeoman UI"

Reply:
Avital saw it and it was ok for now. It will be changed once they decide how final titles should look like.

Reply:
We agreed to extract "Yeoman UI Project Generators" to a wrapper extension in a separate PR.